inspirehep/utils/record.py

```diff
@@ -107,6 +109,15 @@ def getitem(k, v, default):
     return value
 
 
+def get_record_type(record):
+    """Get record's type e.g. 'literature', 'jobs' etc."""
+    schema = record.get('$schema')
+    if schema:
+        return InspireRecordIdProvider.schema_to_pid_type(schema)
+    else:
+        return None
```

Comment:
Exception is probably the best, because it needs to be handled, so the caller knows that they should be careful.